### PR TITLE
Updating docs handbook (draft 1)

### PR DESCRIPTION
**Pull request by jimangel**
This is a continuation of: https://github.com/kubernetes/sig-release/pull/316
Resolves: https://github.com/kubernetes/sig-release/issues/192

This is a WIP and the [sig-docs 1.14 release team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/release_team.md) will modify throughout the release and finalize it after 1.14 drops.

/cc @justaugustus @zacharysarah

### CODE REVIEW

File: release-team/role-handbooks/docs/README.md [hunk 1]
```diff
@@ -1,75 +1,112 @@
...
+[Existing sig-release sig-docs' docs](https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks/docs)
+[Existing sig-release handbooks]()
 
-SIG Docs is part of the Kubernetes release process: its [maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers) support documentation created/updated for releases and published on the [kubernetes website](https://kubernetes.io/).
+### Common Terms
 
-The designated "release meister" (aka Docs Lead), voluntold from [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers), is responsible for managing the docs release  for new and updated documentation [kubernetes.io](https://kubernetes.io/) during quarterly releases.
+| Variable       | Explaination | Example |
+| -------------- | -------- | -------- |
+| <current release>| Active Kubernetes release     | 1.13     |
```
Comment (jimangel) [inline release-team/role-handbooks/docs/README.md]: need to escape the brackets.

**Comment by justaugustus** [comment]
@jimangel -- would prefer we merge an update, prior to the end of the release cycle.

**Comment by jimangel** [comment]
@justaugustus give me a date and you got it!

**Comment by justaugustus** [comment]
Spoke w/ @jimangel. He'll plan to have something solid up for next week.

**Comment by justaugustus** [comment]
Thanks for the updates, @jimangel!
/lgtm
/approve

**Pull request merged**